templates/data/kyma-integration-data.yaml

```diff
@@ -101,7 +101,7 @@ templates:
                   name: "pre-main-kyma-integration-k3s-compass-dev"
                   # following regexp won't start build if only Markdown files were changed
                   run_if_changed: "^((tests/fast-integration\\S+|resources\\S+|installation\\S+|tools/kyma-installer\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))"
-                  optional: true
+                  optional: false
```

Comment:
You can remove this key. It's not needed if you want to make job required.

Reply:
Done